data/_main.cfg

```diff
@@ -49,22 +49,24 @@
 #endif
 [/ais]
 
+#ifdef DEBUG_MODE
+# load tests that need manual user input to run and validate
+{test/scenarios/manual_tests}
```

Comment:
I'd like to change the controlling `#ifdef` back to `#ifdef TEST` so that the `GENERIC_UNIT_TEST` and `COMMON_KEEP` macros can be used in manual tests. Also, these tests current need `-d` in addition to `-t testname` on the command line.

Would changing this back break something else?

Reply:
It shouldn't.

Reply:
Can't you just delete this block? Isn't it included in the `{test}` inclusion further down under `#ifdef TEST`?

Reply:
It isn't, since it was done as part of the "make all manual test scenarios be accessible in the same way" change, which then ended up getting partially reverted because I guess people like using `-t` rather than `-u` for certain tests.

Reply:
That's the entire purpose of `-t` though – to run a test scenario that requires interaction. The manual test scenarios under `test/` aren't even all of them; there are more such scenarios in the `ai/` directory. It keeps them out of the create game screen while also providing a nice convenient list if you bind a key to the "Run Test Scenario" action.

Reply:
The point being that we have some manual tests that require using `-t`, some that require using `-u`, and then the elevation test scenario that's available by default along with the rest of the normal MP scenarios. It doesn't make any sense.

Reply:
Are there really manual tests that require using `-u`? To begin with, any test that runs with `-u` will also run with `-it`, but if there are any manual tests that include `is_unit_test=yes` then that should be removed, and then they'll also show up in the list in the UI.

From the sounds of it, the elevation test scenario should probably be using `[test]` instead of `[multiplayer]`. If there are any other test scenarios that show up like that, the same argument applies.

Reply:
`test_max_menu_items` has `is_unit_test = yes`. I could've sworn there were more, but apparently not.